package.json

```diff
@@ -13,7 +13,7 @@
     "build": "npm run build:common && npm run build:js",
     "build-uncompressed": "npm run build:common && npm run build:js-uncompressed",
     "cypress": "run-p --race server cypress:test",
-    "cypress:install": "npm install --no-save cypress@^3.1.5 @percy/cypress@^2.0.0",
+    "cypress:install": "npm install --no-save cypress@3.6.1 @percy/cypress@^2.0.0",
```

Comment:
Is this good or we want a new PR to update Cypress to latest version?

Reply:
this should be fine, but if we're bumping this, let's use 3.7.0 instead.

Reply:
fixed